README.md

```diff
@@ -1,14 +1,61 @@
-# XAITK - Saliency
+![xaitk-logo](./docs/figures/xaitk-wordmark.png)
 
-## Intent
-Provide interfaces that convey a standard API for visual saliency
-map generation.
+<hr/>
+
+[![Documentation Status](https://readthedocs.org/projects/xaitk-saliency/badge/?version=latest)](https://xaitk-saliency.readthedocs.io/en/latest/?badge=latest)
+[![badge-unittests](https://github.com/xaitk/xaitk-saliency/actions/workflows/ci-unittests.yml/badge.svg)](https://github.com/XAITK/xaitk-saliency/actions/workflows/ci-unittests.yml)
+[![badge-notebooks](https://github.com/xaitk/xaitk-saliency/actions/workflows/ci-example-notebooks.yml/badge.svg)](https://github.com/XAITK/xaitk-saliency/actions/workflows/ci-example-notebooks.yml)
+[![codecov](https://codecov.io/gh/XAITK/xaitk-saliency/branch/master/graph/badge.svg?token=VHRNXYCNCG)](https://codecov.io/gh/XAITK/xaitk-saliency)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=XAITK_xaitk-saliency&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=XAITK_xaitk-saliency)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/XAITK/xaitk-saliency.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/XAITK/xaitk-saliency/context:python)
+
+The `xaitk-saliency` package is an open-source, explainable AI framework for
```

Comment:
Define XAI here since we use the acronym below. We might want to also update our Github repo description to match (not strictly necessary).

```suggestion
The `xaitk-saliency` package is an open-source, explainable AI (XAI) framework for
```

Reply:
Sounds good. Added.

Reply:
"open source" is not hyphenated per the [Kitware Style Guide](https://docs.google.com/document/d/1YasFqNoaQrjC_xijluTpg6_4cgu3dWAp8x02tAdLIG0/edit) (not the xaitk style sheet, since this guideline applies to Kitware text in general)